man/notes/handling-introspection.R

```diff
@@ -73,7 +72,6 @@ foo(error = handle)
 #> >7.   └─global h(simpleError(msg, call))
 #>  8.     └─global handler_helper()
 
-
```

Comment:
oh yeah I gave up on this issue when I implemented double spacing... Double lines between comments are squeezed down to one.